performance_manager/lib/rt_vehicle_positions.py

```diff
@@ -290,17 +274,18 @@ def process_vehicle_positions(db_manager: DatabaseManager) -> None:
 
     # check metadata table for unprocessed parquet files
     paths_to_load = get_unprocessed_files(
-        "RT_VEHICLE_POSITIONS", db_manager.get_session()
+        "RT_VEHICLE_POSITIONS", db_manager, file_limit=6
     )
 
     process_logger.add_metadata(paths_to_load=len(paths_to_load))
 
-    for folder, folder_data in paths_to_load.items():
+    for folder_data in paths_to_load:
+        folder = str(pathlib.Path(folder_data["paths"][0]).parent)
         ids = folder_data["ids"]
```

Comment:
Implement new `get_unprocess_files` function. Limit number of paths to process to 6, so this should only be 6 hours with of parquet files at a time.